epub34/authoring/index.html

```diff
@@ -1246,23 +1256,31 @@ <h3>Exempt resources</h3>
 							document=]); <strong><em>and</em></strong></p>
 					</li>
 					<li>
-						<p>it is not embedded directly in EPUB content documents (e.g., via [[?html]]&#160;[=embedded
-							content=] and [[?svg]]&#160;<a href="https://www.w3.org/TR/SVG/embedded.html#ImageElement"
-									><code>image</code></a> and <a
-								href="https://www.w3.org/TR/SVG/embedded.html#ForeignObjectElement"
-									><code>foreignObject</code></a> elements).</p>
+						<p>it is not directly rendered in its native format in an EPUB content document.</p>
 					</li>
 				</ul>
 
 				<p>This exemption allows EPUB creators to include resources in the [=EPUB container=] that are not for
-					use by EPUB reading systems. The primary case for this exemption is to allow data files to travel
-					with an [=EPUB publication=], whether for scripts to use in their constituent EPUB content documents
-					or for external applications to use (e.g., a scientific journal might include a data set with
-					instructions on how to extract it from the EPUB container).</p>
-
-				<p>It also allows EPUB creators to use foreign resources in foreign content documents without reading
-					systems or [=EPUB conformance checkers=] having to understand the fallback capabilities of those
-					resources (i.e., the requirement for a fallback for the foreign content document covers any
+					use by EPUB reading systems. Resources that benefit from this exemption include:</p>
+
+				<ul>
+					<li>script code modules, such as WebAssembly [[?wasm-core-2]];</li>
+					<li>script inputs, such as fetched [[?fetch]] data or images (refer to <a
+							href="#confreq-cd-scripted-flbk"></a> for restrictions on dynamically adding such resources
+						to EPUB content documents);</li>
+					<li>data files for use by external applications (e.g., a scientific journal might include a data set
+						with instructions on how to extract it from the EPUB container).</li>
+				</ul>
+
+				<div class="note">
+					<p>An example of resource not exempted by this rule would be a image referenced by a <a
+							data-cite="css-values#urls"><code>url</code> function</a> [[css-values]] in a CSS style
+						sheet, as this would not result in core media type resource being displayed.</p>
```

Comment:
I am having a little trouble with the double negatives ("not exempt"/"would not result"), but I have no specific suggestion to make it clearer.

Reply:
I've tweaked this a bit in the last commit and added the additional context of it being a foreign image resource.